compute/apiv1/smoke_test.go

```diff
@@ -0,0 +1,290 @@
+	ctx := context.Background()
+	c, err := NewSecurityPoliciesRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	globalCLient, err := NewGlobalOperationsRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defaultDescription := "default rule"
+	description := "test rule"
+	defaultPriority  := int32(2147483647)
+	priority := int32(0)
+	action := "allow"
+
+	srcIpRanges := []string{
+		"*",
+	}
+	config := &computepb.SecurityPolicyRuleMatcherConfig{
+		SrcIpRanges: srcIpRanges,
+	}
+	versionExpr := computepb.SecurityPolicyRuleMatcher_SRC_IPS_V1
+	matcher := &computepb.SecurityPolicyRuleMatcher{
+		Config: config,
+		VersionedExpr: &versionExpr,
+	}
+	securityPolicyRule := &computepb.SecurityPolicyRule{
+		Action: &action,
+		Priority: &priority,
+		Description: &description,
+		Match: matcher,
+	}
+	securityPolicyRuleDefault := &computepb.SecurityPolicyRule{
+		Action: &action,
+		Priority: &defaultPriority,
+		Description: &defaultDescription,
+		Match: matcher,
+	}
+
+	rules := []*computepb.SecurityPolicyRule{
+		securityPolicyRule,
+		securityPolicyRuleDefault,
+	}
+
+	securityPolicy := &computepb.SecurityPolicy{
+		Name: &name,
+		Rules: rules,
+	}
+
+	insertRequest := &computepb.InsertSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicyResource: securityPolicy,
+	}
+	insert, err := c.Insert(ctx, insertRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+
+	waitGlobalRequest := &computepb.WaitGlobalOperationRequest{
+		Project: projectId,
+		Operation: insert.GetName(),
+	}
+	_, err = globalCLient.Wait(ctx, waitGlobalRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	fmt.Printf("Inserted security policy named %s\n", name)
+	defer ForceDeleteSecurityPolicy(name, ctx, c)
+
+	removeRuleRequest := &computepb.RemoveRuleSecurityPolicyRequest{
+		Priority: &priority,
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+
+	rule, err := c.RemoveRule(ctx, removeRuleRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	waitGlobalRequestRemove := &computepb.WaitGlobalOperationRequest{
+		Project: projectId,
+		Operation: rule.GetName(),
+	}
+	_, err = globalCLient.Wait(ctx, waitGlobalRequestRemove)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+
+	getRequest := &computepb.GetSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	get, err := c.Get(ctx, getRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+	assert.Equal(t, 1, len(get.GetRules()))
+
+	deleteRequest := &computepb.DeleteSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	_, err = c.Delete(ctx, deleteRequest)
+	if err != nil {
+		fmt.Println(err)
+		t.Error(err)
+	}
+}
+
+func ForceDeleteSecurityPolicy(name string, ctx context.Context, client *SecurityPoliciesClient){
+	deleteRequest := &computepb.DeleteSecurityPolicyRequest{
+		Project: projectId,
+		SecurityPolicy: name,
+	}
+	_, err := client.Delete(ctx, deleteRequest)
+	if err != nil {}
```

Comment:
Empty conditional. Should this be a t.Helper?

Reply:
I removed empty conditional, we want to swallow error here